users/vieting/experiments/switchboard/ctc/feat/fullsum_ctc_raw_samples.py

```diff
@@ -159,7 +161,42 @@ def make_conformer_fullsum_ctc_model(
     if recognition:
         python_code = []
     else:
-        if specaug_old is not None:
+        if specaug_stft is not None:
+            frame_size = specaug_stft.pop("frame_size", 400)
+            frame_shift = specaug_stft.pop("frame_shift", 160)
+            fft_size = specaug_stft.pop("fft_size", 512)
```

Comment:
I would argue we should use the same STFT settings as for log Mel by default
```suggestion
            frame_size = specaug_stft.pop("frame_size", 200)
            frame_shift = specaug_stft.pop("frame_shift", 80)
            fft_size = specaug_stft.pop("fft_size", 256)
```